app/src/main/java/com/stl/skipthelibrary/DatabaseHelper.java

```diff
@@ -50,7 +50,7 @@ public void onComplete(@NonNull Task<AuthResult> task) {
                             Toast.makeText(context, "Authentication Succeeded.", Toast.LENGTH_SHORT).show();
                             FirebaseUser firebaseUser = firebaseAuth.getCurrentUser();
                             accountCreated(new User(firstName + " " + lastName, userName, firebaseUser.getUid()
-                                    , new ContactInfo(emailAddress, phoneNumber, null)));
+                                    , new ContactInfo(emailAddress, phoneNumber, null), image));
                         }
                         else {
                             Toast.makeText(context, "Registration failed: " + task.getException().getMessage(), Toast.LENGTH_SHORT).show();
```

Comment:
Please take out this line, the user object makes the intent way to fat it seems - https://github.com/CMPUT301W19T09/SkipTheLibrary/pull/24/files#diff-9f561655c6a3a052da8bf6530fc6b741L114

Reply:
Done and Done